samples/v1.6/Elements/Carousel.Header.Vertical.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
I think we have too many vertical carousel samples. I would make sure each sample demonstrate specific functions of the vertical Carousel.